lib/src/widgets/chat/ds_file_message_bubble.widget.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/controllers/chat/ds_file_message_bubble.controller.dart';
+import 'package:blip_ds/src/widgets/loading/ds_circle_loading.widget.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get_state_manager/get_state_manager.dart';
+
+class DSFileMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final String url;
+  final int size;
+  final String filename;
+  final DSFileMessageBubbleController controller;
+
+  DSFileMessageBubble(
+      {super.key,
+      required this.align,
+      required this.url,
+      required this.size,
+      required this.filename})
+      : controller = DSFileMessageBubbleController();
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () => controller.openFile(filename, url),
```

Comment:
Speak with our PD to check if this tapping action will be enabled across the entire bubble or just when tapping the icon.

Reply:
the action is in all bubble, confirmed by PD